oci/client.go

```diff
@@ -27,6 +27,8 @@ import (
 // Client interface used by SpecOpt
 type Client interface {
 	SnapshotService(snapshotterName string) snapshots.Snapshotter
+
+	Namespace() string
```

Comment:
Rather than expanding the interface, create an interface in the file it is used and check for it there, maybe like  `type namespaced interface { Namespace() string }`. Since the function is only used in one place and not required in the implementations, for example there could be an implementation which is not namespaced.

Reply:
Define a Namespace interface in the file, and let the client implement it, return the default namespace, is this what it means?  

“for example there could be an implementation which is not namespaced. ”What does this sentence mean, is to implement a Namespace interface that returns empty?

Reply:
If you solved the issue in the client package, then no need for a new function or interface

Reply:
The `Namespace interface` is no longer introduced, and whether to inject `Client Default Namespace` is determined by judging whether the user has set it.